src/osgPlugins/nvtt/NVTTImageProcessor.cpp

```diff
@@ -4,7 +4,9 @@
 
 #include <nvtt/nvtt.h>
 #include <string.h>
-
```

Comment:
We should double check that NVTT project is building without the 3.4 makOsg changes to the cmakelists.txt file.